examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/CloseSessionCommand.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ *   Copyright (c) 2023 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package com.matter.controller.commands.pairing
+
+import chip.devicecontroller.ChipDeviceController
+import com.matter.controller.commands.common.CredentialsIssuer
+import com.matter.controller.commands.common.MatterCommand
+import java.util.concurrent.atomic.AtomicInteger
+import java.util.concurrent.atomic.AtomicLong
+
+class CloseSessionCommand(controller: ChipDeviceController, credsIssuer: CredentialsIssuer?) :
+  MatterCommand(controller, "close-session", credsIssuer) {
```

Comment:
Consider converting the base classes `Command` and `MatterCommand` first? This then allows you to use named arguments to clarify some of the literals:

```
class CloseSessionCommand(...) :
  MatterCommand(controller, commandName = "close-session", credsIssuer) {

  init {
    addArgument(
      name = "destination-id",
      min = 0,
      max = Long.MAX_VALUE,
      out = destinationId,
      desc = null,
      optional = false,
    )
  }
}
```

Reply:
I hit some compiling issue if I convert the Java files under Common, so I choose to first convert the upper layer files and then the dependent Java files .

Registered https://github.com/project-chip/connectedhomeip/issues/25496 to track this comment